### DISCUSSION TITLE
Acerca do Hash na integração de documentos manuais

### DISCUSSION THREAD

**Discussion by AfonsoMenata** [assoft-portugal/SAF-T-AO#124] [Q&A] [answered]
Bom dia, pessoal. Preciso uma vez mais de vossa ajuda!

Para criar a assinatura de um documento emitido manualmente e integrado na aplicação, deve-seadicionar uma informação:
1-FTM ab/001, neste formato. Segundo o ponto 8 dos requisitos de validaçãom alínea e).

A minha questão é, onde adicionar essa informação no hassh?? Concatenando, com ";" antes??

Agredeço a vossa apreciação.

**Comment by cryptolopes** [comment]
@AfonsoMenata Podes carregar os requisitos para aqui?

**Comment by AfonsoMenata** [comment]
@cryptolopes. Está em pdf, não consigo anexar. Trata-se do decreto presidencial que fala sobre os rquisitos de validação dos softwares.

**Comment by AfonsoMenata** [comment]
@cryptolopes.
http://www.ucm.minfin.gov.ao/cs/groups/public/documents/document/zmlu/mdu1/~edisp/minfin055809.pdf

**Comment by cryptolopes** [comment, marked as answer]
O elemento `HashControl` deve ser preenchido de acordo com a regra evidenciada no Ponto 8.
A Assinatura de um documento recuperado é na prática irrelevante e não conheço muitos casos em que isso se faça, mas não passará pela recriação da assinatura, mas pela geração de uma nova na respetiva secção de recuperação. O elemento `HashControl` não é tido em consideração para o `Hash`.

**Comment by AfonsoMenata** [comment]
Muito obrigado pelo contributo. Desejo muito sucesso!

Em qui., 4 de fev. de 2021 às 19:06, Nelson <notifications@github.com>
escreveu:
